dianna/methods/lime_tabular.py

```diff
@@ -63,6 +63,7 @@ def __init__(
             LimeTabularExplainer, kwargs)
 
         # temporary solution for setting num_features and top_labels
+        # when fixed, also fix in dashboard Tabular.py -> _feature_names
```

Comment:
Is it fixed in the next PR?

Reply:
No, the temporary fix itself was already here, it has been added in [PR661](https://github.com/dianna-ai/dianna/pull/661/files). I did not try to fix the general problem

Reply:
I believe [issue 801](https://github.com/dianna-ai/dianna/issues/801) mentions the general issue